### PR TITLE
Studio: Add AI guidelines link

### PR DESCRIPTION
**Pull request by kozer**
Fixes https://github.com/Automattic/dotcom-forge/issues/7847
Related  https://github.com/Automattic/dotcom-forge/issues/7446

## Proposed Changes

This PR adds AI guidelines URL to studio

## Testing Instructions
1. Navigate to AI assistant tab
2. Ensure that you see a guidelines section like so:
2024-06-20T13:39:35,499593815+03:00

4. Ensure that clicking it, navigates you to https://automattic.com/ai-guidelines/

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

**Comment by matt-west** [comment]
@kozer Can you please update this so it's more in-keeping with the app design. Here's an [updated design](https://www.figma.com/design/RToz6tIuQ7nlZrikBte4GU/Studio?node-id=3274-115973&t=v7AHDY34CFrccMg8-4) to work from. (Feel free to ping me when things like this come up in future.)

image

Here's how it looks when the chat has scrolled beneath the prompt input:

image

File: src/components/content-tab-assistant.tsx [hunk 1]
```diff
@@ -198,15 +199,23 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
...
+					disabled={ disabled }
+					input={ input }
+					setInput={ setInput }
+					handleSend={ () => handleSend() }
+					handleKeyDown={ handleKeyDown }
+					clearInput={ clearInput }
+					isAssistantThinking={ isAssistantThinking }
+				/>
+				<div data-testid="guidelines-link" className="text-gray-500 self-end p-2">
+					{ createInterpolateElement( __( 'Powered by exerimental AI. <a>Learn More.</a>' ), {
```
Comment (matt-west) [inline src/components/content-tab-assistant.tsx]: There’s a typo in experimental.

```suggestion
					{ createInterpolateElement( __( 'Powered by experimental AI. <a>Learn More.</a>' ), {
```

**Review by matt-west — DISMISSED** [review]

**Comment by sejas** [comment]
Let's add the blur effect in a separate issue: https://github.com/Automattic/dotcom-forge/issues/7678

**Review by wojtekn — APPROVED** [review]
Works well, and the code change looks clear.

@kozer Let's fix the PR description and solve the conflict before we merge.

File: src/components/ai-input.tsx [hunk 2]
```diff
@@ -99,66 +99,58 @@ export const AIInput = ( {
...
-									<span className="ltr:pl-2 rtl:pl-2 text-red-600">
-										{ __( 'Clear conversation' ) }
-									</span>
-								</MenuItem>
-							</MenuGroup>
-						</>
-					) }
-				</DropdownMenu>
-			</div>
+			</DropdownMenu>
```
Comment (kozer) [inline src/components/ai-input.tsx:153]: Note: It seems confusing, but the only thing to note here is that I removed the outer div, and moved it up to the parent.

File: src/components/content-tab-assistant.tsx (hunk 1 above)
Comment (kozer) [reply src/components/content-tab-assistant.tsx]: Changed as requested! Thanks for finding this!

**Comment by kozer** [comment]
@matt-west , @sejas will create a new ticket, so we can track all the needed changes to the input and tackle them holistically. I changed a bit though, so we it's closer to designs ( missing the blur part, as it was a bit tricky to achieve, and it's not tightly related to this issue )

**Comment by katinthehatsite** [comment]
Nice work!!

I have noticed one small thing - the background of the assistant icon and the dropdown menu seem to have light grey color while the rest of the textarea remains white:

Screenshot 2024-06-20 at 1 05 43 PM

File: src/components/content-tab-assistant.tsx [hunk 3]
```diff
@@ -252,15 +253,33 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
...
+						disabled={ disabled }
+						input={ input }
+						setInput={ setInput }
+						handleSend={ () => handleSend() }
+						handleKeyDown={ handleKeyDown }
+						clearInput={ clearInput }
+						isAssistantThinking={ isAssistantThinking }
+					/>
+					<div data-testid="guidelines-link" className="text-a8c-gray-50 self-end p-2">
+						{ createInterpolateElement( __( 'Powered by experimental AI. <a>Learn More</a>' ), {
```
Comment (katinthehatsite) [inline src/components/content-tab-assistant.tsx]:
```suggestion
						{ createInterpolateElement( __( 'Powered by experimental AI. <a>Learn more</a>' ), {
```

Comment (kozer) [reply src/components/content-tab-assistant.tsx]: Thanks! Fixed!

**Comment by kozer** [comment]
> Nice work!!
> 
> I have noticed one small thing - the background of the assistant icon and the dropdown menu seem to have light grey color while the rest of the textarea remains white:
> 
> Screenshot 2024-06-20 at 1 05 43 PM

Thank you Kat! Not is should be ok! :smile:

File: src/components/tests/content-tab-assistant.test.tsx [hunk 4]
```diff
@@ -86,6 +88,13 @@ describe( 'ContentTabAssistant', () => {
 		expect( textInput.placeholder ).toBe( 'What would you like to learn?' );
 	} );
 
+	test( 'renders guideline section', () => {
+		render( <ContentTabAssistant selectedSite={ runningSite } /> );
+		const guideLines = getGuidelinesLink();
+		expect( guideLines ).toBeInTheDocument();
+		expect( guideLines ).toHaveTextContent( 'Powered by experimental AI. Learn more' );
+	} );
```
Comment (sejas) [inline src/components/tests/content-tab-assistant.test.tsx:96]: Thanks for adding the tests!

**Review by sejas — APPROVED** [review]
Looks great! I've removed the right space to align it with the end of the input.

studio-powered-by-ai

**Pull request merged**